src/pages/index.js

```diff
@@ -1,12 +1,13 @@
 import "./index.css";
-
-import { initialCards, validationSettings } from "../utils/constants.js";
+import Api from "../components/Api.js";
+import { validationSettings } from "../utils/constants.js";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
```

Comment:
```js

      this._formElement.reset();
      this._disableButton();
```

Please, delete this code from lines 84 and 85 in `FormValidator.js`


You need to **reset** the form only   in block `then` in order not to do that immediately after `submit` because when there is an error, the modal will still be opened and the user can try to send the data again to the server without having to type the data into the inputs once more.


---------------

You need to **disable** the button only in block `then` after a **successful response** to be able to click it again if there is a server error.